themes/digital.gov/src/scss/new/shortcodes/_note.scss

```diff
@@ -1,139 +1,146 @@
+    margin-top: 0;
+    padding-left: 20px;
 
     li {
-      @include u-margin-bottom(2);
+      margin-bottom: 16px;
     }
   }
-}
-
-// For notes within list items
-li .note {
-  @include u-padding-y("05");
-  @include u-padding-x(1);
-}
-
-// Activities  — color settings
-.note.activity {
-  @include u-bg("orange-warm-5");
-  @include u-text("orange-warm-60");
-  @include u-border-left("05", "solid", "orange-warm-30");
-
-  a {
-    @include u-text("orange-warm-60");
-    @include u-border-bottom("1px", "solid", "orange-warm-60");
-
-    &:hover {
-      @include u-text("orange-warm-80");
-    }
+  // For notes within list items
+  li .note {
+    @include u-padding-y("05");
+    @include u-padding-x(1);
   }
-
-  .highlight {
-    @include u-bg("orange-warm-10");
-  }
-}
-
-// Activities  — color settings
-.note.alert {
-  @include u-bg("red-5");
-  @include u-text("red-60v");
-  @include u-border-left("05", "solid", "red-40v");
-
-  a {
-    @include u-text("red-60v");
-    @include u-border-bottom("1px", "solid", "red-60v");
-
-    &:hover {
-      @include u-text("red-80v");
+  // Activities  — color settings
+  &--activity {
+    background-color: color("orange-warm-5");
+    border-left-color: color("orange-warm-30");
+    border-left-style: solid;
+    border-left-width: 5px;
+    color: color("orange-warm-60");
+
+    a {
+      border-bottom-color: color("orange-warm-60");
+      border-bottom-style: solid;
+      border-bottom-width: 1px;
+      color: color("orange-warm-60");
+
+      &:hover {
+        color: color("orange-warm-80");
+      }
     }
-  }
 
-  .highlight {
-    @include u-bg("red-10");
+    .highlight {
+      background-color: color("orange-warm-10");
+    }
   }
-}
-
-// Comment — color settings
-.note.comment {
-  @include u-bg("blue-cool-10");
-  @include u-text("blue-cool-70");
-  @include u-border-left("05", "solid", "blue-cool-50");
+  // Alert styles
+  &--alert {
+    background-color: color("red-5");
+    border-left-color: color("red-40v");
+    border-left-style: solid;
+    border-left-width: 5px;
+    color: color("red-60v");
+
+    a {
+      border-bottom-style: solid;
+      border-bottom-width: 1px;
+      color: color("red-60v");
+
+      &:hover {
+        color: color("red-80v");
+      }
+    }
 
-  h4 {
-    i {
-      @include u-text("blue-cool-60");
+    .highlight {
+      background-color: color("red-10");
     }
   }
+  // Comment — color settings
+  &--comment {
+    background-color: color("blue-cool-10");
+    border-left-color: color("blue-cool-50");
+    border-left-style: solid;
+    border-left-width: 5px;
+    color: color("blue-cool-80");
+
+    a {
+      border-bottom-style: solid;
+      border-bottom-width: 1px;
+
+      &:hover {
+        color: color("blue-cool-80");
+      }
+    }
 
-  a {
-    @include u-text("blue-cool-70");
-    @include u-border-bottom("1px", "solid", "blue-cool-50");
+    .highlight {
+      background-color: color("white");
+    }
```

Comment:
Confirming is this correct? The legacy styles had:

```scss
  .highlight {
    @include u-bg("white");
    @include u-bg("yellow-5");
  }
```

Which sets background to white and then overrides again as `yellow-5`. This looks like a bug, so trying to confirm what the correct value should be.